.env.example

```diff
@@ -0,0 +1,5 @@
+BASE_URL=http://localdev.hackney.gov.uk
+HACKNEY_JWT_SECRET=sekret
+HACKNEY_COOKIE_NAME=hackneyToken
+REACT_APP_ENV
```

Comment:
You'll need to make sure these env vars are added to the deployment, the secret is already there (I think) so the others can be added to serverless.yml